.golangci.yml

```diff
@@ -42,6 +43,25 @@ linters-settings:
     - go.uber.org/atomic: 'use "sync/atomic" instead'
   misspell:
     locale: US
+  gci:
+    # Section configuration to compare against.
```

Comment:
Compare against what?

Reply:
You need to ask the author https://golangci-lint.run/usage/linters/#gci
I'll remove the comment.